engine/src/main/scala/cromwell/server/CromwellDeadLetterListener.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.server
+
+import akka.event.DeadLetterListener
+
+class CromwellDeadLetterListener extends DeadLetterListener {
+  def shutdownReceive: Receive = {
+    // This silences the dead letter messages when Cromwell is shutting down
+    case _ if CromwellShutdown.shutdownInProgress() =>
```

Comment:
I'm curious what these messages are and who's sending to whom.

Can we do some gentler printing (maybe don't mention the word "dead", perhaps "unhandled" or even "bonus" :) ).

I had actually started a branch to do this myself: https://github.com/broadinstitute/cromwell/compare/db_dead_letters#diff-ef805a742940b60e9d750303111fbb71R64

Reply:
Would logging them at debug level be ok ? I fear that even if we re-word the message people will get confused..

Reply:
sure!